ui/backend/server/server/settings_mini.py

```diff
@@ -34,7 +38,19 @@ def get_from_env(
 
 HAMILTON_AUTH_MODE = "permissive"
 
-ALLOWED_HOSTS = ["localhost", "backend", "0.0.0.0", "127.0.0.1"]
+# set up allowed hosts
+try:
+    hostname = socket.gethostname()
+    local_ip = socket.gethostbyname(hostname)
+    hosts_to_add = [local_ip, hostname]
+except Exception:
+    logger.exception("Could not get hostname or local IP")
```

Comment:
I know you copied from me but this will look like an error to anyone on a mac for when it doesn't work. Maybe just a logger.warning.